deploy/helm/secret-operator/templates/roles.yaml

```diff
@@ -67,11 +104,13 @@ rules:
       - podlisteners
     verbs:
       - get
+{{ if .Capabilities.APIVersions.Has "security.openshift.io/v1" }}
   - apiGroups:
     - security.openshift.io
     resourceNames:
-    - privileged
+      - stackable-secret-operator-scc
```

Comment:
Seems to have mixed indentation styles compared to the lines around it, but maybe that's just github mobile.

Reply:
Also, I think the PR description should explain how this is different to the other operators (using the rootv2 thing)

Reply:
I fixed the indentation thanks.

There is a [high-level note ](https://docs.stackable.tech/home/nightly/kubernetes/openshift#_security_context_constraints) in the documentation describing the different SCCs. Do you think it's enough for now?

Reply:
FYI, other sections have different style indentations for lists